config/initializers/devise.rb

```diff
@@ -20,7 +20,6 @@
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
   config.mailer_sender = "please-change-me-at-config-initializers-devise@example.com"
-
```

Comment:
checkout file này

Reply:
file này e có config 1 chút nên k checkout được ạ

Reply:
Toàn thấy xóa dòng thôi mà